mobify/platform-scaffold
# Rename test folder from 'system' to 'e2e'

We generally use the term 'e2e' to refer to our end-to-end tests.

## Changes
- Rename all instances of 'system' to 'e2e'
- Update nightwatch-commands to 3.0.0 which allows us to drop `site.js` and make the `.preview` command more explicit

## How to test-drive this PR
- Rebuild on CircleCI and verify tests still run and reports are generated and saved to the correct places
https://circleci.com/gh/mobify/platform-scaffold/tree/rename-test-folders
